castle/extractors/ip.py

```diff
@@ -9,6 +9,11 @@ def call(self):
         return self.wsgi_request_ip()
 
     def wsgi_request_ip(self):
+        cf_ip = self.request.environ.get('HTTP_CF_CONNECTING_IP')
```

Comment:
I had to lookup what this header is.  I would add a comment indicating what it is

Reply:
Valid point. I plan to extend README with all the info